addons/message_whats_new_v2.22/manifest.json

```diff
@@ -21,7 +21,35 @@
       {
         "id": "c_1-1",
```

Comment:
@flodolo Does this message ID need to change since the old string has already been uploaded to pontoon after the last PR was merged?

Reply:
Yes, all string changes require new IDs.